.github/workflows/create-javadoc-and-tag.yml

```diff
@@ -1,6 +1,11 @@
 name: Create Javadoc and Tag
 on:
   workflow_dispatch:
+    inputs:
+      tags:
```

Comment:
we plural "tags"? We only expect a single tag name, so "tag" or even "tagName" would be better?